Makefile

```diff
@@ -127,6 +128,12 @@ override CFLAGS += -mthumb -mthumb-interwork -O2 -mabi=apcs-gnu -mtune=arm7tdmi
 ifeq ($(ANALYZE),1)
 override CFLAGS += -fanalyzer
 endif
+# Only throw an error for unused elements if its RH-Hideout's repo
+ifeq ($(UNUSED_ERROR),0)
+ifneq ($(GITHUB_REPOSITORY_OWNER),rh-hideout) 
```

Comment:
Do we need this if we manually set `UNUSED_ERROR: 1` in the actions workflow?

Reply:
The `ifneq ($(GITHUB_REPOSITORY_OWNER),rh-hideout)` part? Yes, because otherwise workflows for cloned remote repos will fail for users otherwise. Failed test assumptions also fail in our repo but not others to avoid their workflows fail in case they change data from the default. It would be annoying for them that building works locally but not on CI.